Assets/MixedRealityToolkit.SDK/Experimental/Dialog/Scripts/DialogButtonType.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Dialog
+{
+    /// <summary>
+    /// Enum describing the style (caption) of button on a Dialog.
+    /// </summary>
+    [Flags]
+    public enum DialogButtonType
+    {
+        None = 0,
```

Comment:
Consider adding summary comments for these enum values. They will help make the MRTK documentation best in class. Thanks!